amfGrpcServer/amf_server.go

```diff
@@ -0,0 +1,82 @@
+package amf_server
```

Comment:
this file should be moved to main package.
and change the directory name amfGrpcServer--> protos

Reply:
Resolved. Moved this file to service package.